rabbitmq/init.sls

```diff
@@ -1,8 +1,5 @@
 {% from "rabbitmq/package-map.jinja" import pkgs with context %}
 
```

Comment:
Uh Please move this to the end of the file for users who are doing include: rabbitmq. It wont break your use